lib/jsonapi/fetchdata/collection.rb

```diff
@@ -16,21 +16,21 @@ def initialize scope, *parsers
       end
 
       def find conditions={}
-        process @klass, @adapter.parameters(conditions)
+        process(@adapter.parameters(conditions))
       end
 
       def process conditions
         conditions.each do |k, v|
           @scope = case k.to_sym
-            when :include then @scope.includes(*v).join(*v)
```

Comment:
One of the starting points of making this gem was to standardize eager loading of relationship objects.
Seems I need to use includes(:relation).references(:relation) instead.
.join which should've been .join*s* anyway.
Is there any reason this would be unwanted behaviour in Rails 3?